src/js/port_handler.js

```diff
@@ -21,7 +21,10 @@ PortHandler.initialize = function () {
     const portPickerElementSelector = "div#port-picker #port";
     this.portPickerElement = $(portPickerElementSelector);
     this.selectList = document.querySelector(portPickerElementSelector);
-    this.initialWidth = this.selectList.offsetWidth + 12;
+    this.initialWidth = this.selectList.offsetWidth;
+
+    ConfigStorage.get('showVirtualMode', res => self.showVirtualMode = res.showVirtualMode);
+    ConfigStorage.get('showAllSerialDevices', res => self.showAllSerialDevices = res.showAllSerialDevices);
```

Comment:
Didn't we move to sync api now? 
```suggestion
    self.showVirtualMode = ConfigStorage.get('showVirtualMode').showVirtualMode);
    self.showAllSerialDevices = ConfigStorage.get('showAllSerialDevices').showAllSerialDevices;
```

Reply:
Yes here: https://github.com/betaflight/betaflight-configurator/pull/2636 (will do afterwards when rebasing).

Reply:
:partying_face: Finally got it approved. Rebased now.